src/com/lftechnology/java/training/krishna/basics/evenfibonacci/EvenFibonacciDemo.java

```diff
@@ -0,0 +1,31 @@
+package com.lftechnology.java.training.krishna.basics.evenfibonacci;
+
+import java.util.logging.Logger;
+
+/**
+ * EvenFibonacciDemo.java
+ * 
+ * Log the output on the console.
+ * 
+ * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+ * @version 1.0
+ */
+
+public class EvenFibonacciDemo {
+	private final static Logger LOGGER = Logger.getLogger(EvenFibonacciDemo.class.getName());
+	
+	/**
+	 * This is the main method which makes use of sumEvenFibonacci() method.
+	 * 
+	 * @param args Unused.
+	 * @return Nothing.
+	 * @author Krishna Timilsina <krishnatimilsina@lftechnology.com>
+	 */
+	public static void main(String[] args) {
+		EvenFibonacci evenFibonacci = new EvenFibonacci(0, 1);
+		int fib = evenFibonacci.sumEvenFibonacci();
+		LOGGER.info("sum of even fibonacci number : "  + fib);
```

Comment:
use parameterised logger if you are dealing with the variable.
Have a look at : http://stackoverflow.com/questions/15880629/java-logging-with-multiple-parameters